sql/parser/walk.go

```diff
@@ -242,3 +248,52 @@ func WalkStmt(v Visitor, stmt Statement) {
 		}
 	}
 }
+
+// GetParameters returns the Params slice as a `parameters`.
+func GetParameters(r driver.Request) Parameters {
+	return Parameters(r.Params)
+}
+
+// Parameters implements the Args interface.
+type Parameters []driver.Datum
+
+// Arg implements the Args interface.
+func (p Parameters) Arg(name string) (Datum, bool) {
```

Comment:
moved to sql. It was moved to sql/driver to keep it near all wire datum  related code.